pkg/http/api_test.go

```diff
@@ -2894,14 +2898,14 @@ func TestV1Alpha1DistributedLock(t *testing.T) {
 		b, _ := json.Marshal(&req)
 
 		resp := fakeServer.DoRequest("POST", apiPath, b, nil)
-		assert.Equal(t, 500, resp.StatusCode)
+		assert.Equal(t, 400, resp.StatusCode)
```

Comment:
should this be `403 Forbidden` instead?

Reply:
Agree not 400, but maybe 409 (Conflict)

Reply:
Isn't 409 more related to version conflicts? 
In this case it's someone else owning the lock right?